### DISCUSSION TITLE
Stable version on `release` page

### DISCUSSION THREAD

**Discussion by shiroinekotfs** [LSPosed/MagiskOnWSALocal#617] [Q&A]
Hi MagiskOnWSALocal.

For each stable version, why don't you create a release on the `release` page?

It will help users install the stable version, and roll back to the older version easier.

**Comment by Doddddd** [comment]
Refer to opengapps.
https://opengapps.org/blog/post/2019/08/23/sourceforge-migration

**Comment by Howard20181** [comment]
https://github.com/LSPosed/MagiskOnWSALocal/issues/4#issuecomment-1376508507
